sdk/keyvault/keyvault-certificates/review/keyvault-certificates.review.api.md

```diff
@@ -0,0 +1,286 @@
+    // Warning: (ae-forgotten-export) The symbol "Certificate" needs to be exported by the entry point index.d.ts
+    listCertificates(options?: RequestOptionsBase): PagedAsyncIterableIterator<Certificate, Certificate[]>;
+    listCertificateVersions(name: string, options?: RequestOptionsBase): PagedAsyncIterableIterator<Certificate, Certificate[]>;
+    listDeletedCertificates(options?: KeyVaultClientGetDeletedCertificatesOptionalParams): PagedAsyncIterableIterator<DeletedCertificate, DeletedCertificate[]>;
+    mergeCertificate(name: string, x509Certificates: Uint8Array[], options?: RequestOptionsBase): Promise<Certificate>;
+    readonly pipeline: ServiceClientOptions;
+    purgeDeletedCertificate(name: string, options?: RequestOptionsBase): Promise<null>;
+    recoverDeletedCertificate(name: string, options?: RequestOptionsBase): Promise<Certificate>;
+    restoreCertificate(certificateBackup: Uint8Array, options?: RequestOptionsBase): Promise<Certificate>;
+    setCertificateContacts(contacts: Contact[], options?: RequestOptionsBase): Promise<Contacts>;
+    setCertificateIssuer(issuerName: string, provider: string, options?: KeyVaultClientSetCertificateIssuerOptionalParams): Promise<CertificateIssuer>;
+    updateCertificate(name: string, version: string, options?: KeyVaultClientUpdateCertificateOptionalParams): Promise<Certificate>;
+    updateCertificateIssuer(issuerName: string, options?: KeyVaultClientUpdateCertificateIssuerOptionalParams): Promise<CertificateIssuer>;
+    updateCertificatePolicy(name: string, policy: CertificatePolicy, options?: RequestOptionsBase): Promise<CertificatePolicy>;
+    readonly vaultBaseUrl: string;
+}
+
+// @public
+export interface Contact {
+    emailAddress?: string;
+    name?: string;
+    phone?: string;
+}
+
+// @public
+export interface Contacts {
+    contactList?: Contact[];
+    readonly id?: string;
+}
+
+// @public
+export interface DeletedCertificate extends Certificate {
+    readonly deletedDate?: Date;
+    recoveryId?: string;
+    readonly scheduledPurgeDate?: Date;
+}
+
+// @public
+export interface ErrorModel {
+    readonly code?: string;
+    readonly innerError?: ErrorModel;
+    readonly message?: string;
+}
+
+// @public
+export interface IssuerAttributes {
+    readonly created?: Date;
+    enabled?: boolean;
+    readonly updated?: Date;
+}
+
+// @public
+export interface IssuerCredentials {
+    accountId?: string;
+    password?: string;
+}
+
+// @public
+export interface IssuerParameters {
+    certificateTransparency?: boolean;
+    certificateType?: string;
+    name?: string;
+}
+
+// @public
+export interface KeyProperties {
+    curve?: JsonWebKeyCurveName;
+    exportable?: boolean;
+    keySize?: number;
+    keyType?: JsonWebKeyType;
+    reuseKey?: boolean;
+}
+
+// @public
+export interface KeyVaultClientGetCertificateIssuersOptionalParams extends coreHttp.RequestOptionsBase {
```

Comment:
Should extend OperationOptions once https://github.com/Azure/azure-sdk-for-js/pull/5708 lands

Reply:
ListPropertiesOfCertificateIssuersOptions?